modules/gitops-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-4_{context}"]
+= Release notes for {gitops-title} 1.8.4
+
+{gitops-title} 1.8.4 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="new-features-1-8-4_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.6.13.
+
+[id="fixed-issues-1-8-4_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, Argo CD was becoming unresponsive when there was an increase in namespaces and applications. This update fixes the issue by removing a deadlock. Deadlock occurs when two functions are competing for resources. Now, you should not experience crashes or unresponsiveness when there is an increase in namespaces or applications. link:https://issues.redhat.com/browse/GITOPS-3192[GITOPS-3192]
+
+* Before this update, the Argo CD application controller resource could suddenly stop working when resynchronizing applications. This update fixes the issue by adding logic to prevent a cluster cache deadlock. Now, you should not experience the deadlock situation, and applications should resynchronize successfully. link:https://issues.redhat.com/browse/GITOPS-3052[GITOPS-3052]
```

Comment:
```suggestion
* Before this update, the Argo CD application controller resource could suddenly stop working when resynchronizing applications. This update fixes the issue by adding logic to prevent a cluster cache deadlock. Now, applications should resynchronize successfully. link:https://issues.redhat.com/browse/GITOPS-3052[GITOPS-3052]
```